ui/src/utils/mod.rs

```diff
@@ -16,26 +16,31 @@ pub mod lifecycle;
 pub fn get_available_themes() -> Vec<Theme> {
     let mut themes = vec![];
 
-    for file in WalkDir::new(&STATIC_ARGS.themes_path)
-        .into_iter()
-        .filter_map(|file| file.ok())
-    {
-        if file.metadata().unwrap().is_file() {
-            let theme_path = file.path().display().to_string();
-            let pretty_theme_str = get_pretty_name(&theme_path);
-            let pretty_theme_str = titlecase(&pretty_theme_str);
-
-            let styles = fs::read_to_string(&theme_path).unwrap_or_default();
-
-            let theme = Theme {
-                filename: theme_path.to_owned(),
-                name: pretty_theme_str.to_owned(),
-                styles,
-            };
-
-            themes.push(theme);
+    let mut add_to_themes = |themes_path| {
+        for file in WalkDir::new(themes_path)
+            .into_iter()
+            .filter_map(|file| file.ok())
+        {
+            if file.metadata().unwrap().is_file() {
```

Comment:
Even though this been here prior to the PR, best to be cautious about unwrapping